Decred Wallet/Features/Backup/BackupVerifyViewController.swift

```diff
@@ -106,29 +99,24 @@ extension ConfirmNewWalletSeedViewController: UITableViewDelegate, UITableViewDa
     }
     
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        let cell = tableView.dequeueReusableCell(withIdentifier: "tripleSeedCell", for: indexPath) as? SeedConfirmTableViewCell
+        let cell = tableView.dequeueReusableCell(withIdentifier: "tripleSeedCell", for: indexPath) as? BackupVerifyTableViewCell
         
         let userSelection = self.selectedWords[indexPath.row]
         cell?.setup(num: indexPath.row, seedWords: seedWordsGroupedByThree[indexPath.row], selectedWord: userSelection)
         
         cell?.onPick = {(index, seedWord) in
+            self.banner?.dismiss()
             self.selectedWords[indexPath.row] = seedWord
             
-            self.btnConfirm.isEnabled = self.selectedWords.reduce(true, { (res, input) -> Bool in
-                return res && input != ""
-            })
+            var allChecked = true;
+            for seedIndex in 0...32 {
```

Comment:
Let's use the count of the seeds instead of the number 32. In the future, if we reduce/increase the number of seeds it will necessitate someone to change this.